guide/guide.html

```diff
@@ -746,14 +746,14 @@ <h4>clone and init</h4> New objects are made by cloning existing ones. A clone i
                     <pre>
 myDog name := "rover"
 myDog sit := method("I'm sitting\n" print)
-</pre> When an object is cloned, its "init" slot will be called if it has one.
+</pre>
```

Comment:
This is potentially more contentious. There was duplication here between:
* `When an object is cloned, the new instance's init slot will be activated which gives the object a chance to initialize itself.`
* When an object is cloned, its "init" slot will be called if it has one.

I attempted to merge the two and removed the latter. That said, `if it has one` was lost in my edit.